spec/controllers/course/assessment/question/programming_controller_spec.rb

```diff
@@ -7,6 +7,13 @@
     let(:user) { create(:user) }
     let(:course) { create(:course, creator: user) }
     let(:assessment) { create(:assessment, course: course) }
+    let(:question_programming_attributes) do
+      attributes_for(:course_assessment_question_programming).
+        slice(:title, :description, :maximum_grade, :language, :memory_limit,
+              :time_limit).tap do |result|
+          result[:language_id] = result.delete(:language).id
+        end
```

Comment:
`end` at 15, 8 is not aligned with `attributes_for(:course_assessment_question_programming).` at 11, 6 or `:time_limit).tap do |result|` at 13, 14.